src/gui/categoryfiltermodel.cpp

```diff
@@ -435,9 +433,9 @@ void CategoryFilterModel::populate()
         else
         {
             new CategoryModelItem(
-                        m_rootItem, category
+                        m_rootItem, categoryName
                         , std::count_if(torrents.begin(), torrents.end()
-                                        , [category](Torrent *torrent) { return torrent->belongsToCategory(category); }));
+                                        , [categoryName](Torrent *torrent) { return torrent->belongsToCategory(categoryName); }));
```

Comment:
(optional) maybe use a local variable for `std::count_if` since the code is long.

Reply:
> (optional) maybe use a local variable for `std::count_if` since the code is long.

Ok.